examples/experimental/gentoo.yaml

```diff
@@ -0,0 +1,8 @@
+images:
+- location: "https://distfiles.gentoo.org/experimental/amd64/openstack/gentoo-openstack-amd64-default-latest.qcow2"
+  arch: "x86_64"
+
+# The built-in containerd installer does not support Gentoo currently.
```

Comment:
Should this be just `plain: true` ?
I guess a lot of config is still incompatible with Gentoo?

Reply:
The guestagent seems to work, though.